SYCL/Plugin/sycl-ls.cpp

```diff
@@ -1,5 +1,5 @@
 // RUN: sycl-ls --verbose | grep "Device \[" | wc -l >%t.verbose.out
-// RUN: sycl-ls | wc -l >%t.concise.out
+// RUN: sycl-ls | grep "^\[[a-z0-9_]*:[a-z]*:[0-9]*\]\ " | wc -l >%t.concise.out
```

Comment:
what is the weird filter doing and can we please avoid it?

Reply:
With the ESIMD_EMULATOR support, `sycl-ls` application prints out like below.
```
EMU: *** Warning no kernel information query method is supported in current build.
EMU: [kernel support,critical] kernel data query via debug data is not supported in current build.
EMU: [kernel support] *** Warning kernel  arguments data is not found.
[ext_intel_esimd_emulator:gpu:0] Intel(R) ESIMD_EMULATOR/GPU, ESIMD_EMULATOR 1.0.7-CM_EMU [0.1.0]
[host:host:0] SYCL host platform, SYCL host device 1.2 [1.2]
```

CM_EMU library supporting ESIMD_EMULATOR could print out extra info for debugging - like first three lines of above. It cannot be controlled from SYCL as it is from CM_EMU and CM_EMU prints out them by default. This change is to filter out such extra info for comparison between outputs from 'verbose' output and 'concise' output.

There is ongoing internal discussion for controlling printouts from CM_EMU. As soon as it is decided how these printouts are controlled and applied in CM_EMU, I'm going to remove this filtering.